docker/.env.example

```
@@ -0,0 +1,139 @@
+############
+
+PGRST_DB_SCHEMAS=public,storage,graphql_public
+
+
+############
+# Auth - Configuration for the GoTrue authentication server.
+############
+
+## General
+SITE_URL=http://localhost:3000
+ADDITIONAL_REDIRECT_URLS=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ADDITIONAL_REDIRECT_URLS key should go before the SITE_URL key

(UnorderedKey)